mlserver-huggingface/tox.ini

```diff
@@ -36,6 +37,18 @@ commands =
     # run rock tests
     pytest -vvv --tb native --show-capture=all --log-cli-level=INFO {posargs} {toxinidir}/tests
 
+[testenv:sanity]
+passenv = *
+deps =
+    juju~=2.9.0
```

Comment:
Should we bump this?  I think we've moved to something >3

Reply:
Kept this as is in order to bump it during ROCKs update. In theory, these are still CKF 1.7 ROCKs.

Reply:
oh yeah good point.  there's so many little things we can trip up on in here :/